drake/doc/code_style_guide.rst

```diff
@@ -28,17 +31,15 @@ We now strictly follow the `Google C++ Style Guide <https://google.github.io/sty
 * No dynamic allocation in the inner simulation/control loops.  Code should be still be thread-safe (e.g. be careful with pre-allocations).
 * Classes and methods should be documented using [doxygen](https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html).
 * Embrace templates/C++11 when it makes the code more correct (more clear or more readable also implies more correct).  Minimize template requirements on public interfaces.  Avoid explicit template instantiations in cc files when possible.
+* For the `order of includes <https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes>`_ style rule, surround Drake header files with double quotes and all other header files with angle brackets. Follow the guide as best you can in terms of arranging the ``#include`` statements in the following order: (1) Related header, (2) C library, (3) C++ library, (4) other libraries' .h, (5) your project's .h. Then accept whatever :ref:`clang-format <code-style-tools-clang-format>` enforces.
+
```

Comment:
The (1)..(5) list appears to duplicate google-styleguide exactly, and so this list is brittle with respect to changes in that document (it that document gains additional specific advice, or changes its mind, Drake should follow suit).